packages/react-brand-designer/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@trycourier/react-brand-designer",
+  "version": "1.17.0",
+  "description": "Emendable brand designer for multi-channel notifications",
+  "author": "Drew Youngwerth <drew@youngwerth.com>",
+  "homepage": "https://github.com/trycourier/courier-react/tree/main/packages/react-elements#readme",
+  "license": "ISC",
```

Comment:
maybe MIT?

Reply:
Inbox uses ISC. I like ISC a little more. Its essentially the same license as MIT but cleaner.